modules/db_instance/versions.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">= 1.0"
+  required_version = ">= 1.11.1"
```

Comment:
this would be considered a breaking change, plus a large number of our users will not be on the bleeding edge